common/src/snapshot/mark_set_go.rs

```diff
@@ -77,11 +82,10 @@ pub trait SnapshotsCallback {
 pub struct Snapshot {
     /// snapshot_stake: stake distribution map (credential -> lovelace amount)
     pub snapshot_stake: VMap<StakeCredential, i64>,
-    // snapshot_delegations: delegation map (credential -> stake pool key hash)
-    // pub snapshot_delegations: VMap<Credential, StakePool>,
-
-    // snapshot_pool_params: pool parameters map (stake pool key hash -> pool params)
-    //  snapshot_pool_params: VMap<Vec<u8>, Vec<u8>>,
+    // snapshot_delegations: vmap<credential, key_hash<stake_pool>>,)
+    pub snapshot_delegations: VMap<StakeCredential, Hash<28>>,
```

Comment:
I think this could use `PoolId` instead, given it's a map of staking credential to the pool to which they're delegated to.


```suggestion
    pub snapshot_delegations: VMap<StakeCredential, acropolis_common::PoolId>,
```